back/app/services/xlsx_service.rb

```diff
@@ -173,6 +173,11 @@ def generate_initiatives_xlsx(initiatives, view_private_attributes: false)
       { header: 'author_name',          f: ->(i) { format_author_name i } },
       { header: 'author_email',         f: ->(i) { i.author&.email } },
       { header: 'author_id',            f: ->(i) { i.author_id } },
+      { header: 'cosponsors',           f: lambda { |i|
+                                             i.cosponsors_initiatives.map do |ci|
+                                               "{ id: #{ci.user_id}, email: #{i.cosponsors.find { |user| user.id == ci.user_id }.email}, status: #{ci.status} }"
+                                             end.join(',')
+                                           }, skip_sanitization: true },
```

Comment:
I agree, `email` may be useful in some cases, but storing multiple fields in one cell is not very common. And this format `{ id: ...},{ id: ...}` is very unusual too. It's not CSV, not JSON.

As the combination of cosponsors and this exporter won't be probably used much, maybe very simple version would be enough
```suggestion
      { header: 'cosponsors',           f: ->(i) { i.cosponsors_initiatives.map { |ci| ci.user.email }.join(',') } },
```